autofit/database/model/prior.py

```diff
@@ -57,15 +63,23 @@ class PriorModel(Object):
     A prior model
     """
 
-    __tablename__ = "prior_model"
+    try:
+
+        from sqlalchemy import Column, Integer, ForeignKey
+
+        __tablename__ = "prior_model"
 
-    id = Column(
-        Integer,
-        ForeignKey(
-            "object.id"
-        ),
-        primary_key=True,
-    )
+        id = Column(
+            Integer,
+            ForeignKey(
+                "object.id"
+            ),
+            primary_key=True,
+        )
+
+    except ModuleNotFoundError:
```

Comment:
alias sqlalchemy and then conditionally import that?

Reply:
The problem is because these are class attributes methods like:

```
id = Column(
            Integer,
            ForeignKey(
                "object.id"
            ),
            primary_key=True,
        )
```

Are called when autofit is imported. So they need to be in try / excepts. I think.